cmd/dbbrowser/main.go

```diff
@@ -1,19 +1,28 @@
 package main
 
 import (
+	"flag"
+	"fmt"
 	"log"
 	"net/http"
 	"os"
 
 	"github.com/medyagh/gopogh/pkg/db"
 )
 
+var dbPath = flag.String("db_path", "", "path to cloudsql db in the form of 'host=HOST_NAME user=DB_USER dbname=DB_NAME password=DB_PASS'")
+
 func main() {
-	path := "host=k8s-minikube:us-west1:flake-rate user=postgres dbname=flakedbdev password="
-	path = path + os.Getenv("DB_PASS")
+	flag.Parse()
+	if *dbPath == "" {
+		fmt.Println("db_path not specified. defaulting to minikube")
```

Comment:
lets not have a default for minikube here, gopogh is a separate project from minikube